cmd/serve.go

```diff
@@ -161,6 +162,8 @@ func ServeRunner(do *definitions.Do) func(*cobra.Command, []string) {
 				DefaultConfigFilename)
 		}
 
+		vm.SetDebug(do.Debug)
```

Comment:
ugh. that is a very direct way to pull this through the whole stack... but  you might be right that for now it is a useful things to have.